compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.7'
```

Comment:
shouldn't this file be called `docker-compose.yml`?

Reply:
it works both ways